korean/coco/coco.md

```diff
@@ -1,6 +1,7 @@
 핫초코를 사러 나온 당신... 돈은 있나요? 핫초코를 어디서 살건가요? 
 
-[돈이 없으면 뭘 어쩌겠어? 다시 방으로 돌아간다](../marshmallow.md) 집에서 잠이나 자!
```

Comment:
However GitHub marked this pull request as having no conflicts with the base branch, can't it be auto-merged? I didn't delete any line but switch the order of lines.